### ISSUE TITLE
Support incremental addition to a note with org-roam-protocol

### ISSUE THREAD

**Issue by zaeph** [org-roam/org-roam#1240]
Follow-up to #1239.

Once the PR is merged, `org-roam-protocol` will support passing the webpage's selection to the note by having `%i` in the template (specified in `org-roam-capture-ref-templates`).  This opens a path of improvement where we could incrementally add content to a note by repeatedly running a bookmarklet on an associated page.

Compare the position of `%i` in the two following templates:
```el
(setq org-roam-capture-ref-templates
      '(("r" "ref" plain
         #'org-roam-capture--get-point
         ""
         :file-name "web/${slug}"
         :head "#+title: ${title}\n%i"
         :unnarrowed t)

        ("i" "incremental" plain
         #'org-roam-capture--get-point
         "%i"
         :file-name "web/${slug}"
         :head "#+title: ${title}\n"
         :unnarrowed t)))
```

In `r`, `%i` is in the `:head` section of the template, which means that it will only be expanded once upon creating the file.  In `i`, `%i` is in the body of the template, which means that it will be expanded every time the template is called.

Right now, this is buggy, since including `:empty-lines-before 1` in the `i` template results in the empty line being inserted *before* `:head` when the template is used to create a file.  However, I think this would be a good idea which would allow for basic incremental reading (see [SuperMemo's article on incremental reading](https://www.supermemo.com/en/archives1990-2015/help/read) for the full picture).

**Comment by AtomicNess123** [comment]
Has this been fixed?

**Comment by jethrokuan** [comment]
5db30d8fdc21b61b06afb12179a8b8f0297e69e5 fixes this, but only in v2.

**Comment by AtomicNess123** [comment]
You mean, we must reinstall org-roam?

**Comment by jethrokuan** [comment]
You can either choose to wait, or [upgrade to v2's alpha](https://github.com/org-roam/org-roam/wiki/Hitchhiker%27s-Rough-Guide-to-Org-roam-V2)

**Comment by AtomicNess123** [comment]
I did package-reinstall org-roam. Is this how I am to upgrade?

**Comment by jethrokuan** [comment]
Now that v2 is merged into master you can expect MELPA to build the package within a day or two. Then running `package-reinstall` should get you the version with the fix.
